hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/AlertsEngineImpl.java

```diff
@@ -455,44 +459,47 @@ public void removeTrigger(String tenantId, String triggerId) {
     }
 
     private void removeTrigger(Trigger trigger) {
+        final String tenantId = trigger.getTenantId();
+        final String triggerId = trigger.getId();
+
+        // If necessary, clean up working memory
```

Comment:
Although I am sure it is not a problem from the perspective of rule firings, I will make the change to modify() because that seems the appropriate construct.